<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmCadCarroDialog</class>
 <widget class="QDialog" name="FrmCadCarroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro Carro</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ano</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edNome">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edMarca">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edTipo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edAno">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btGravar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gravar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btLimpar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancelar">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>18</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Cadastro de Carros</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
